SimTKcommon/include/SimTKcommon/internal/String.h

```diff
@@ -418,6 +424,13 @@ template <> inline
 bool tryConvertStringTo(const String& value, double& out)
 {   return value.tryConvertToDouble(out); }
 
+// Specialization to ensure recognition of non-finite values NaN, Inf, etc.
+template <> inline
+bool tryConvertStringTo(const String& value, Recorder& out)
+{
```

Comment:
Can you use the same brackets style as the other specializations? Thanks

Reply:
Ok, done